js-apps/meep-frontend/src/js/containers/meep-container.js

```diff
@@ -27,7 +27,7 @@ import * as meepSandboxCtrlRestApiClient from '../../../../../js-packages/meep-s
 import * as meepMonEngineRestApiClient from '../../../../../js-packages/meep-mon-engine-client/src/index.js';
 import * as meepGisEngineRestApiClient from '../../../../../js-packages/meep-gis-engine-client/src/index.js';
 
-import MeepDrawer from './meep-drawer';
```

Comment:
Has the meep-drawer.js file been removed? If we no longer need it we should remove it along with any other dead code that may come from this change.

Reply:
Removed unused code